spec/features/dashboard_spec.rb

```diff
@@ -41,6 +41,24 @@
       expect(page).to have_content(help2.user.email)
       expect(page).to_not have_content(zone.user.email)
     end
+
+    scenario 'can filter for feeling lucky' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.